charts/kuberpult/values.yaml.tpl

```diff
@@ -64,7 +64,8 @@ rollout:
 
 ingress:
   create: true
-  annotations: {}
+  annotations:
+    nginx.ingress.kubernetes.io/proxy-read-timeout: 300
```

Comment:
should this be a helm chart parameter?

Reply:
Good question. I think this the least bad way to do this. We could write it directly into the ingress template, but then it would conflict if somebody sets a value for this annotation directly. We could also merge the customer supplied annotations and our default annotations in the template and then render that new value. This would however be less obvious because  we would have more logic in the templates. I think this way has least frictions. It's completely obivious where the value comes from and how to change it.